tests/lib/hardware/libvirt.py

```diff
@@ -46,6 +46,8 @@
 
 logger = logging.getLogger(__name__)
 
+# this lock needs to resolve race condition issue while defining a node
+libvirtDefineNodeLock = threading.Lock()
```

Comment:
no camel case iirc. and libvirt is not needed. maybe just `node_lock` ?

Reply:
I would prefer to leave _libvirt_ prefix since it is related to libvirt only bug. Also, libvirt is using camel, that is why I used camel   to emphasize libvirt only relation.

Reply:
Besides we the *node_lock* is not good name for this lock, because we do not lock the node, we even do not lock node spin up or boot, we only need to isolate node definition which is very cheap operation.

Reply:
we don't use camel case here. so please change it.